.github/workflows/terratests-prod-suite-all.yaml

```diff
@@ -29,6 +29,7 @@ jobs:
       TEST_DATA_PROD_VIRTUAL_DEVICE_2_WAN_CONNECTION: ${{ secrets.TEST_DATA_PROD_VIRTUAL_DEVICE_2_WAN_CONNECTION }}
       TEST_DATA_PROD_VIRTUAL_DEVICE_2_AZURE_CONNECTION: ${{secrets.TEST_DATA_PROD_VIRTUAL_DEVICE_2_AZURE_CONNECTION}}
       TEST_DATA_PROD_VIRTUAL_DEVICE_2_PORT_CONNECTION: ${{secrets.TEST_DATA_PROD_VIRTUAL_DEVICE_2_PORT_CONNECTION}}
+      TEST_DATA_PROD_CLOUD_ROUTER_2_VIRTUAL_DEVICE_CONNECTION: ${{secrets.TEST_DATA_PROD_CLOUD_ROUTER_2_VIRTUAL_DEVICE_CONNECTION}}
```

Comment:
Keep the test names next to their neighbors. Same with the `jq and echo` statements as well. That way it's clear to see the list with a quick glance without spending more time.

Reply:
Addressed with [this](https://github.com/equinix/terraform-equinix-fabric/pull/111/commits/2b8d064506143b64d097a00e3a7e8d2d272dd140) commit